priv/fake/courses.txt

```diff
@@ -0,0 +1,138 @@
+Mestrado em Bioquímica Aplicada
+Mestrado em Biotecnologia
+Mestrado em Ciência Política
+Mestrado em Ciências Criminais
+Mestrado em Ciências da Comunicação
+Mestrado em Ciências da Educação
+Mestrado em Ciências da Saúde
+Mestrado em Ciências e Tecnologias do Ambiente
+Mestrado em Cognição Humana
+Mestrado em Comunicação de Ciência
+Mestrado em Comunicação, Arte e Cultura
+Mestrado em Construção e Reabilitação Sustentáveis
+Mestrado em Contabilidade
+Mestrado em Crime, Diferença e Desigualdade
+Mestrado em Desafios das Cidades
+Mestrado em Design de Produto e Serviços
+Mestrado em Design e Inovação de Produtos Têxteis e Acessórios
+Mestrado em Design para a Comunicação de Moda
+Mestrado em Direito Administrativo
+Mestrado em Direito da União Europeia
+Mestrado em Direito das Crianças, Família e Sucessões
+Mestrado em Direito dos Contratos e da Empresa
+Mestrado em Direito dos Negócios, Europeu e Transnacional
+Mestrado em Direito e Informática
+Mestrado em Direito Judiciário
+Mestrado em Direito Transnacional da Empresa e das Tecnologias Digitais
+Mestrado em Direito Tributário
+Mestrado em Direitos Humanos
+Mestrado em Economia
+Mestrado em Economia Industrial e da Empresa
+Mestrado em Economia Monetária, Bancária e Financeira
+Mestrado em Economia Social
+Mestrado em Educação
+Mestrado em Educação Pré-Escolar
+Mestrado em Educação Pré-Escolar e Ensino do 1º Ciclo do Ensino Básico
+Mestrado em Engenharia Aeroespacial
+Mestrado em Engenharia Biomédica
+Mestrado em Engenharia Civil
+Mestrado em Engenharia de Estruturas
+Mestrado em Engenharia de Materiais
+Mestrado em Engenharia de Polímeros
+Mestrado em Engenharia de Sistemas
+Mestrado em Engenharia de Telecomunicações e Informática
+Mestrado em Engenharia do Produto
+Mestrado em Engenharia e Gestão da Qualidade
+Mestrado em Engenharia e Gestão de Operações
+Mestrado em Engenharia e Gestão de Sistemas de Informação
+Mestrado em Engenharia e Gestão Industrial
+Mestrado em Engenharia Eletrónica Industrial e Computadores
+Mestrado em Engenharia Física
+Mestrado em Engenharia Humana
+Doutoramento em Biologia
+Doutoramento em Ciência e Engenharia de Polímeros e Compósitos
+Doutoramento em Ciências da Cultura
+Doutoramento em Ciências da Linguagem
+Doutoramento em Ciências da Literatura
+Doutoramento em Engenharia Biomédica
+Doutoramento em Engenharia Civil
+Doutoramento em Engenharia de Materiais
+Doutoramento em Engenharia de Tecidos, Medicina Regenerativa e Células Estaminais
+Doutoramento em Engenharia Eletrónica e de Computadores
+Doutoramento em Engenharia Mecânica
+Doutoramento em Engenharia Química e Biológica
+Doutoramento em Engenharia Têxtil
+Doutoramento em Filosofia
+Doutoramento em Geologia - Geoconservação, Geologia Ambiental e Recursos Geológicos
+Doutoramento em Gestão e Tratamento de Resíduos
+Doutoramento em Informática
+Doutoramento em Matemática
+Doutoramento em Química Aplicada
+Doutoramento em Sustentabilidade do Ambiente Construído
```

Comment:
Where was this data obtained from?

Reply:
https://www.uminho.pt/PT/ensino/oferta-educativa/paginas/licenciaturas-e-mestrados-integrados.aspx

https://www.uminho.pt/PT/ensino/oferta-educativa/Paginas/Mestrados.aspx

https://www.uminho.pt/PT/ensino/oferta-educativa/Paginas/Doutoramentos.aspx